tests/test_namespaces.py

```diff
@@ -0,0 +1,55 @@
+from beet.contrib.vanilla import Vanilla
+from beet.contrib.unknown_files import UnknownAsset, UnknownData
+import tempfile
+
+from beet.core.cache import Cache
+from beet.library.base import LATEST_MINECRAFT_VERSION
+from beet.toolchain.helpers import run_beet
+from zipfile import ZipFile
+
+
+def test_namespaces():
+    # Test that all namespaces are known
+    # It loads the vanilla jar and checks
+    # that there are no unknown namespaces
```

Comment:
Shouldn't this be named `test_scopes`? Vanilla only has one namespace (`minecraft`)

Reply:
yes, i will change that